snappy/snapp_test.go

```diff
@@ -476,6 +476,25 @@ func (s *SnapTestSuite) TestUbuntuStoreRepositorySearch(c *C) {
 	c.Check(parts[0].Description(), Equals, "Returns for store credit only.")
 }
 
+func (s *SnapTestSuite) TestUbuntuStoreAll(c *C) {
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		c.Check(r.URL.RawQuery, Equals, "")
+		io.WriteString(w, MockSearchJSON)
+	}))
+	c.Assert(mockServer, NotNil)
```

Comment:
In general if there's no error result we can assume this to never be nil as we do in other cases where we do check the error. Given this is a test-oriented API, most likely it will blow up with a panic if it fails.